tests/tests/lightspeed.spec.ts

```
@@ -127,7 +127,7 @@ test.describe('OLS UI', () => {
 
       await page.goto('/');
 
-      await expect(page.locator(mainButton)).toBeVisible();
+      await expect(page.locator(mainButton)).toBeVisible({ timeout: 30_000});
```

ESLint

[error] Insert `·`

(prettier/prettier)